src/api/app/views/webui2/webui/request/_issues_table.html.haml

```diff
@@ -0,0 +1,26 @@
+%p
+  %b
+    Mentioned Issues (#{issues.length})
+%table
+  %tbody
+    - issues.each do |long_name, issue|
+      - if issue[:owner] && issue[:state]
+        %tr
+          %td
+            = link_to(long_name, issue[:url], class: 'issue_tracker_link')
+          %td.nowrap
+            - if issue[:owner]
+              = user_with_realname_and_icon issue[:owner], short: true
+          %td
+            = issue[:state].capitalize if issue[:state]
+      - else
+        %tr.even
+          %td{ colspan: '3' }
+            = link_to(long_name, issue[:url], class: 'issue_tracker_link')
+      - if issue[:summary]
+        %tr.odd
+          %td{ colspan: '3' }
+            = toggle_sliced_text(issue[:summary])
+-# Stupid XHTML-1.0 strict doesn't like 'target', but all browsers do. Thus, sneak around validation to allow opening a new window upon click:
```

Comment:
eh? :eyes: why are we using `target`?

Reply:
@Ana06:

> eh? :eyes: why are we using `target`?

To open the issue in another browser window. Anyway, this don't need to be done with jQuery, so I removed the jQuery for setting the `target` and put it as an attribute in the `link`.

Reply:
I don't like it anywhere. Why don't we give the user the freedom to decide where to open it? :free: